extensions/BeatsBlox/index.js

```diff
@@ -5888,8 +5888,8 @@
                         return new List(pattern.map((x) => rootNote + x));
                     }),
                     new Extension.Block('setTrackEffect', 'command', 'music','track %supportedEffects effect to %n %', ['Volume','50'], function (effectName, level){
-                        if(parseInt(level) > 100 || level == ''){
-                            throw Error('Level must be a value between 1 and 100');
+                        if(parseInt(level) > 100 || parseInt(level) < 0 || level == '' ||){
```

Comment:
the trailing `||` in this expression is a syntax error and would break the extension

Reply:
Removed it. Thanks for catching that!